vmsyscall/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! syscall serialize/deserialize
+//!
+//! Currently it uses a hard coded page and an I/O trigger.
+//! We might want to switch to MMIO.
+
+#![no_std]
+
```

Comment:
Maybe ` #![deny(missing_docs)]` and add docs to satisfy it?

Reply:
done